security/fma-supervisor.md

```diff
@@ -0,0 +1,314 @@
+    - The Batcher can decline to post beyond the current cross-unsafe head. This will avoid the publishing of bad data so the sequencer may reorg it out, saving the replacement based reorg. If it went on long enough, the Batcher would prevent any new data from being posted to L1, effectively creating a safe-head stall until the sequencer resolved the issue. This *could* be a preferred scenario for some chains.
+    - We need to develop and use production-realistic networks in large scale testing to exercise failure cases and get confidence that the system behaves and recovers as expected.
+- Action Items
+    - Develop Monitoring for any messages where are discovered to be invalid in a block.
+    Operators should be vigilant when this happens.
+    - Additional Monitoring should be developed around ETH transfers per day, or other market behaviors
+    (to be scoped)
+
+
+## FM2b: Supervisor Doesn’t Catch Block Invalidation of Safe Data
+
+- Description
+    - FM2a has occurred, but additionally, the Supervisor doesn’t catch the invalid message when promoting from local-safe to cross-safe.
+    - An output root that builds on this incorrect cross-safe head is published to L1.
+    - At this point, any validators who rely on the Supervisor are following an incorrectly derived chain — blocks between the invalid message and the end of the batch should be replaced by deposit only blocks.
+    - Any validators who *do not* rely on the failing Supervisor will see the correct chain, but there are currently no alternative implementations to use.
+    - An output root posted from this incorrect state would be open to be fault-proven.
+- Risk Assessment
+    - Even Higher Impact, Low Likelihood.
+- Recovery
+    - Within the 12h Safe Head window, if the sequencer is repaired and rewound, it could correctly interpret the batch data to replace a block, and then would have the job of rebuilding the chain from that point. All users would need to upgrade to a version without this derivation bug, and resync the chain from the failed position. Operators who *did not* upgrade and resync would be left on a dead branch that is no longer being updated.
+- Action Items
+    - Improved testing is required across the board for cross-head promotion, as this is the area most likely to
+    break consensus.
+    - Implement an ETH rate limit which prevents more than X Eth from being transferred in a single transaction.
+    This should limit the blast radius of any undiscovered bugs.
+    - Implement a pause mechanism which declines interop transactions from being included in block building.
+        - This pause mechanism should be triggerable via admin API
+        - And also should be responsive to automated Alert/Monitoring based triggers.
+        - Once paused, unpause should be manual to start with.
+    - Optionally: Implement a matching Batcher pause to avoid publishing invalid blocks in batches where avoidable.
```

Comment:
We need to make a determination on this, possible it will be required, but need to analyze further. Approving for now, it gets the gist across.